lighthouse-docker/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM alpine:20210212
+FROM node:18-slim
 
 WORKDIR /opt/lighthouse
 
-RUN apk --update-cache --no-cache \
-     add npm chromium \
-    && npm install lighthouse@8.0.0 \
-    && mkdir -p /root/.config/configstore \
-    && echo '{"isErrorReportingEnabled": false}' > /root/.config/configstore/lighthouse.json
```

Comment:
This config is needed any longer?

Reply:
I suspect the change adding the `--quiet` option/flag does the same?

Reply:
The configuration was set to ensure we always opt out of sending runtime exceptions to Google. Although by default if no response is given it won't send reports.

https://github.com/GoogleChrome/lighthouse/blob/main/docs/error-reporting.md#how-do-i-change-my-opt-in-preference

Reply:
I can re-add it

Reply:
Should we use the `--no-enable-error-reporting` flag?

Reply:
Yeah, I think that would make it more obvious what behavior we actually want.

Reply:
addressed in [`b88d6ad` (#16)](https://github.com/iFixit/lighthouse-docker/pull/16/commits/b88d6adfec316d3be441452bce382f376ba495d7)